src/snippet.rs

```diff
@@ -44,6 +48,14 @@ impl<'a> Group<'a> {
         self
     }
 
+    /// Set the primary [`Level`] for this [`Group`]. If not specified, the
+    /// [`Level`] of the first [`Title`] in the [`Group`] will be used, if no
+    /// [`Title`] is present, it will default to [`Level::ERROR`].
+    pub fn primary_level(mut self, level: Level<'a>) -> Self {
```

Comment:
The following documentation should be updated
https://github.com/rust-lang/annotate-snippets-rs/blob/c195b836696312ee80880c1fe4edf6003222aade/src/snippet.rs#L265-L266